app/templates/_yeoman.gradle

```diff
@@ -15,8 +15,9 @@ task bower(type: Exec) {
 bower.onlyIf { !project.hasProperty('skipBower') }
 processResources.dependsOn 'bower'
 <% }  if(frontendBuilder == 'gulp') {%>
+bower.onlyIf { !project.hasProperty('skipBower') }
 gulp_build.dependsOn 'npmInstall'
 gulp_build.dependsOn 'bower'
-processResources.dependsOn gulp_build
```

Comment:
It does. In ``_prod.gradle`` it depends on ``gulp_build`` so ``processResources`` depends on ``bower`` and ``gulp_build``